parity/rpc_apis.rs

```diff
@@ -627,6 +626,7 @@ impl ApiSet {
 				public_list.insert(Api::ParityAccounts);
 				public_list.insert(Api::ParitySet);
 				public_list.insert(Api::Signer);
+				public_list.insert(Api::SecretStore);
```

Comment:
just a question. Above you've added a comment `SecretStore (UNSAFE: arbitrary hash signing)`, but here `SecretStore` is added to a `SafeContext`. Isn't it contradictory?

Reply:
I was also confused a bit :) If I understood correctly, there are:
1) safe/unsafe APIs
2) safe/unsafe API Contexts
So: SecretStore API is unsafe => it is only enabled when API context is safe. Here's couple of proofs I've found:
1) https://github.com/paritytech/parity/blob/master/parity/rpc_apis.rs#L148 (reveres proof)
2) ParityAccounts API, which is also unsafe, is included to the `SafeContext` API set

Reply:
Yes, indeed that the reasoning, although It's extremely confusing (sorry about that:() Feel free to rename it.